src/.gitignore

```diff
@@ -1,3 +0,0 @@
-target

```

Comment:
Why these are removed, I beliewve we don't want to put target directory to version control

Reply:
This file is not needed.